attack.py

```diff
@@ -102,62 +95,31 @@ def create_parser():
         return parser_obj
 
     @staticmethod
-    def check_req():

```

Comment:
Useless method. If dependencies are not satisfied, the script will fail to run anyway.